libs/status/type-defs/status-payloads/error-info.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @todo
+ */
+export type ErrorInfo = {
+  /**
+   * The reason of the error. This is a constant value that identifies the proximate cause of
+   * the error. Error reasons are unique within a particular domain of errors. This should be at
+   * most 63 characters and match a regular expression of `[A-Z][A-Z0-9_]+[A-Z0-9]`, which
+   * represents UPPER_SNAKE_CASE.
+   */
+  reason: string;
+
+  /**
+   *  The logical grouping to which the "reason" belongs.
+   *  The error domain is typically the registered service name of the tool or product that
+   *  generates the error.
+   */
+  domain: string;
+
+  /**
+   * Additional structured details about this error.
+   *
+   * Keys should match /[a-zA-Z0-9-_]/ and be limited to 64 characters in length. When
```

Comment:
Perhaps it would be useful to mention that the keys are not expected to be `SCREAMING_SNAKE_CASE` in this case but just `camelCase`

Reply:
We can do, I think it doesn't necessarily have to be prescribed (this is the same as Google Cloud's one), and saying it is possibly somewhat superfluous, the examples demonstrate it.